src/scss/modules/sidebar.scss

```diff
@@ -67,7 +68,6 @@
   font-size: 12px;
   line-height: 16px;
   margin-top: 8px;
-  max-width: 232px;
   padding: 8px;
```

Comment:
This is the only place in the app the `.sidebar__info` classname is used. For this clinician sidebar info section.